### PR TITLE
add me

### PR DESCRIPTION
**Pull request by Gitsanto**
Adding omiya

### CODE REVIEW

File: _pins/gitsanto.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+githubHandle:Mario
```
Comment (hectorsector) [inline _pins/gitsanto.json]: @Gitsanto you'll need to include a space after the `:` and your GitHub username here

File: _pins/gitsanto.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+latitude: 35.874987
+longitude: 139.613946
```
Comment (hectorsector) [inline _pins/gitsanto.json]: @Gitsanto the three dashes `---` are needed after line 4, as well.

Comment (Gitsanto) [inline _pins/gitsanto.json]: thank you so much

**Comment by Gitsanto** [comment]
now

**Pull request merged**